### ISSUE TITLE
No burn-in ckpt is provided

### ISSUE THREAD

**Issue by dohyun1411** [nini0919/SemiRES#3]
To run train_semi_CPI.py, it seems we need "./output/1%/refcoco/sup/model_best_refcoco.pth" as a burn-in checkpoint. Where can I get this?

**Comment by nini0919** [comment]
you can get it by training a supervised learning baseline or using our pre-trained burn-in checkpoint for 1% refcoco (https://drive.google.com/file/d/1295BejiOuN8S2pr4MbGlVrvGU_ExChkP/view?usp=drive_link)

**Comment by dohyun1411** [comment]
Thanks for providing the file.
I have one more question: what are the criteria for selecting the burn-in checkpoint? (maybe based on the performance of the validation dataset)

**Comment by nini0919** [comment]
You are correct. Observing the training convergence of the burn-in model on the test set is our method for selecting the the burn-in checkpoint. It can also be noted that the burn-in model does not necessarily need to reach training convergence. In the subsequent mutual learning stage between teacher and student, this labeled data will continue to be trained using student model, so the impact is not significant. Thank you for your question.
